src/main/plugin/iso19139.ca.HNAP/layout/layout-custom-fields.xsl

```diff
@@ -94,177 +94,149 @@
-          <value ref="{gco:CharacterString/gn:element/@ref}" lang="{$metadataLanguage}">
-            <xsl:value-of select="gco:CharacterString"/>
-          </value>
-          &lt;!&ndash;<xsl:message>value main: <xsl:value-of select="gco:CharacterString" /> - <xsl:value-of select="gco:CharacterString/gn:element/@ref" /></xsl:message>&ndash;&gt;
-        </xsl:if>
-
-        &lt;!&ndash; the existing translation &ndash;&gt;
-        <xsl:for-each select="gmd:PT_FreeText/gmd:textGroup/gmd:LocalisedCharacterString">
-          <value ref="{gn:element/@ref}" lang="{substring-after(@locale, '#')}">
-            <xsl:value-of select="."/>
-          </value>
-          &lt;!&ndash;<xsl:message>value alt 1:  <xsl:value-of select="." /> - <xsl:value-of select="gn:element/@ref" /> <xsl:value-of select="substring-after(@locale, '#')" /></xsl:message>&ndash;&gt;
-        </xsl:for-each>
-
-        &lt;!&ndash; and create field for none translated language &ndash;&gt;
-        <xsl:for-each select="$metadataOtherLanguages/lang">
-          <xsl:variable name="currentLanguageId" select="@id"/>
-          <xsl:if test="count($theElement/
-                gmd:PT_FreeText/gmd:textGroup/
-                gmd:LocalisedCharacterString[@locale = concat('#',$currentLanguageId)]) = 0">
-            <value ref="lang_{@id}_{$theElement/parent::node()/gn:element/@ref}"
-                   lang="{@id}"></value>
-
-            &lt;!&ndash;<xsl:message>value alt 1: <xsl:value-of select="$theElement/parent::node()/gn:element/@ref" /> <xsl:value-of select="@id" /></xsl:message>&ndash;&gt;
-
-          </xsl:if>
-        </xsl:for-each>
-      </values>
-    </xsl:variable>
-
-    <xsl:call-template name="render-element">
-      <xsl:with-param name="label"
-                      select="$labelConfig"/>
-      <xsl:with-param name="value" select="$values"/>
-      <xsl:with-param name="cls" select="local-name()"/>
-      <xsl:with-param name="xpath" select="$xpath"/>
-      <xsl:with-param name="directive" select="'gn-organisation-entry-selector-ec'"/>
-      <xsl:with-param name="editInfo" select="gn:element"/>
-      <xsl:with-param name="parentEditInfo" select="../gn:element"/>
-    </xsl:call-template>
-
-  </xsl:template>
+    <xsl:variable name="theElement" select="." />
 
+    <xsl:variable name="hasDefaultValue" select="gco:CharacterString != ''"/>
 
-  <xsl:template mode="mode-iso19139" match="gmd:country" priority="2000">
+    <!--
+      Creates this (actual values of the metadata record);
 
-    <xsl:variable name="xpath" select="gn-fn-metadata:getXPath(.)"/>
-    <xsl:variable name="isoType" select="if (../@gco:isoType) then ../@gco:isoType else ''"/>
-    <xsl:variable name="labelConfig" select="gn-fn-metadata:getLabel($schema, name(), $labels, name(..), $isoType, $xpath)"/>
-
-    <xsl:variable name="theElement" select="." />
+        <values>
+            <value ref="15" lang="eng">Government of Canada; Library of Parliament; david place2</value>
```

Comment:
"david place2" not the best example :)
Here are some [examples from Public Services and Procurement Canada](https://www.tpsgc-pwgsc.gc.ca/apropos-about/rgnstnnll-rgnztnal-eng.html)

Reply:
;)